clients/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
I think this makes sense, but generally speaking we'd tend to have one "fix" per PR for easier reviews. Ditto below.